blogsite/myapp/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth import get_user_model
+
+UserModel = get_user_model()
+
+# Create your models here.
+class Topic(models.Model):
+    topic_name = models.CharField(max_length=64, unique=True)
+    description = models.CharField(max_length=255)
+    users = models.ManyToManyField(UserModel, through='UserTopicRelation')
+
+
+class Article(models.Model):
+    title_article = models.CharField(max_length=255, default='Input your text')
+    text = models.TextField(default='Input your text')
+    created_date = models.DateTimeField(auto_now_add=True)
```

Comment:
```suggestion
    created_at = models.DateTimeField(auto_now_add=True)
```

Бо в Python для праці з часом та датами існують окремі типи даних:

- `date`
- `time`
- `datetime`

Отака назва збиває з пантелику. Бо начебто за логікою там має бути `date`, а насправді `datetime`.